website/views/shared/tasks/lists.jade

```diff
@@ -24,9 +24,14 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
 
           // Add New
           form.task-add(name='new{{list.type}}form', ng-hide='obj._locked', ng-submit='addTask(obj[list.type+"s"],list)')
-            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', required)
+            textarea(rows='6', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', required)
```

Comment:
Let's add this attribute to the textarea:
```
ui-keydown='{"meta-enter":"addTask(obj[list.type+\'s\'], list)"}',
```

This will allow people to submit the tasks by pressing cmd/ctrl + enter.